src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,104 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_LIST_CAPACITY = 10;
+    private static final int HALF_FROM_DEFAULT_LIST_CAPACITY = DEFAULT_LIST_CAPACITY / 2;
```

Comment:
Maybe better don't make constant, but divide and plus it in method, or make constant "1.5" and multiply default
list capacity by it